Automattic/jetpack
# Story block: address minor display issues

Fix minor display issues with the Story block and improve compatibility with themes

#### Changes proposed in this Pull Request:
This PR addresses two display issues with the story block on the frontend:
- [x] CSS and display conflicts: prevent story breaking because of column-count and page-break
- [x] Swap direction of arrow icons for RTL languages

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:
- Apply this patch
- Install and activate the `Cols` theme on your site and publish a story, notice the story isn't split in 2 between the columns
- Go to your site settings, and change the language to an RTL one. Reload a page with a story and try to navigate slides when viewing it fullscreen, notice the left arrow to go to the next slide is on the left as expected
